### DISCUSSION TITLE
mania uses the same color on two columns on argon

### DISCUSSION THREAD

**Discussion by enoslayd** [ppy/osu#20635] [Q&A] [answered]
is it intentional that the two columns on 5k uses pink?
![osu_2022-10-08_22-24-05](https://user-images.githubusercontent.com/41866328/194712425-cde8f5e3-a708-45d8-8838-acb4c2f0ea46.png)

Edit:
also on 6k (pink on rightmost columns)
![osu_2022-10-08_22-27-36](https://user-images.githubusercontent.com/41866328/194712501-9fcd40da-3d73-4b8d-9efc-3fa00487c165.png)

8k (pink on middle)
![osu_2022-10-08_22-28-26](https://user-images.githubusercontent.com/41866328/194712534-13033354-334b-478e-ae47-78f83499697f.png)

9k (yellow on both left and rightmost columns)
![osu_2022-10-08_22-29-03](https://user-images.githubusercontent.com/41866328/194712584-32d0232a-f7bd-4a28-b049-5f44ee823a4c.png)

10k (yellow on left most and 2nd to the right, orange on 2nd left and right most and pink on columns 4 and 5 from the left)
![osu_2022-10-08_22-30-22](https://user-images.githubusercontent.com/41866328/194712625-c1558dbb-a424-42eb-8c69-7777ea45e6d7.png)

**Comment by peppy** [comment, marked as answer]
I implemented these by the design document... which had a colour repeated. My bad, I guess?

**Reply by peppy** [reply]
#20679
